crowdin.yaml

```diff
@@ -41,8 +41,8 @@ files:
         'tr': 'tr'
         'uk': 'uk'
         'vi': 'vi'
-        'zh-CN': 'zh-Hans'
-        'zh-TW': 'zh-Hant'
+        'zh-CN': 'zh-CN'
```

Comment:
I hope this and the changes in website/ don't break docusaurus's own site's current crowdin settings. There's no chinese translation for it right now anyway.